lms/templates/main.html

```diff
@@ -182,3 +182,31 @@
     )
   ) if course_id and enrollment_action else ""
 }</%def>
+
+# Performance beacon for onload times
+% if settings.FEATURES.get('ENABLE_ONLOAD_BEACON', False):
+<script>
+  var sample_rate = ${settings.ONLOAD_BEACON_SAMPLE_RATE};
```

Comment:
The contents of this `script` tag need to be wrapped in an IIFE [1].

Currently, this code will leak several variables into the global
(`window`) namespace (`sample_rate`, `roll`, `onloadBeaconSent`).

This script tag should take the form:
```javascript
(function () {
    var sample_rate = '...';
    // ...
}());
```

[1] https://en.wikipedia.org/wiki/IIFE